platformio.ini

```diff
@@ -44,6 +44,19 @@ build_flags_ha =
 	-DHOME_ASSISTANT_MQTT_USERNAME=\"ard1\"
 	-DHOME_ASSISTANT_MQTT_PASSWORD=\"0\"
 	-DHOME_ASSISTANT_DEVICE_NAME=\"ard1\"
+	; Disable components of the Home Assistant library that we do not use
```

Comment:
@mburumaxwell do you think it's worth providing a link to the documentation describing these macros, e.g.:

https://dawidchyrzynski.github.io/arduino-home-assistant/documents/library/compiler-macros.html#code-optimization

as it's not immediately clear what this is referring to? I'd understand if you don't want to link to a url that could change, but I think it's good to at least make it clear what library this is referring to.

Reply:
Makes lots of sense. Updated with a [permalink](https://github.com/dawidchyrzynski/arduino-home-assistant/blob/1d333ab229b2dcefedf1b51e033f2177b74fca8c/docsrc/source/documents/library/compiler-macros.rst)